src/themes.ts

```diff
@@ -26,6 +26,8 @@ const colorCycle = [
 export const breakpoints: Breakpoints = {
```

Comment:
why there is no `'small'` breakpoint?

Reply:
`small` is not breakpoint, `small` is the screen size from 0 to medium breakpoint.